client/src/components/common/ConfigMenu/ConfigMenuSection.scss

```diff
@@ -5,7 +5,7 @@
   border: 1px solid $lightGrey;
   @include border-radius(6px);
   margin-bottom: 15px;
-  padding: 15px 15px 0 15px;
```

Comment:
@loicsans can I ask whats the purpose of this because its breaking the other sections? I can try implement it just for the vis type section if thats the goal